test/test_nn.py

```diff
@@ -1977,6 +1977,26 @@ def test_orthogonal(self):
                         self.assertEqual(torch.mm(flattened_tensor, flattened_tensor.t()),
                                          torch.eye(rows) * gain ** 2, prec=1e-6)
 
+    def test_PReLU(self):
```

Comment:
Actually it would be better to provide `reference_fn` to the PReLU tests. Grad is checked automatically. Can you please change that?